File: Java/fibonacci.java
import java.util.*;
public class fibonacci 
{
   public static void main(String[] args)
   {
      Scanner sc = new Scanner(System.in);
      System.out.print("Enter the number of numbers you need :");
      int nums = sc.nextInt();
      int prev = 0;
      int curr = 1;
      System.out.println(prev);
      System.out.println(curr);
      for(int i = 0; i<= nums - 3; i++)
      {
         int next_num = prev + curr;
         System.out.println(next_num);
         prev = curr;
         curr = next_num;
      }
      sc.close();
   }   
}

File: C/fibonacci.c
#include <stdio.h>

int main()
{
	int prev_num = 0, next_num = 1,i, counter, temp;
	printf("How many number of the fibonacci series do you want :");
	scanf("%d", &counter);
	printf("\n%d\t", prev_num);
	
	for(i = 1;i < counter; i++)
	{
		printf("%d\t", next_num);
		temp = next_num;
		next_num = prev_num + next_num;
		prev_num = temp;
	}
	return 0;
}
